app/models/user.rb

```diff
@@ -89,7 +97,7 @@ def update_password(params)
     end
 
     errors.add(:password, :empty) if params[:password].blank?
-    errors.add(:password, :password_too_short) if params[:password] && params[:password].strip.length < 6
+    errors.add(:password, :password_too_short) if params[:password] && params[:password].strip.length < 11
```

Comment:
this is confusing. if the length is 10...isn't that a long-enough password? (based on `config.password_length = 10..255`, it implies that 10 is a valid length)

Reply:
The minimum password length should be 10
```suggestion
    errors.add(:password, :password_too_short) if params[:password] && params[:password].strip.length < 10
```